### PR TITLE
@xtina-starr => Bump artsy-passport for redirect bug, and also maintain current filters

### PR DESCRIPTION
**Pull request by mzikherman**
Last bit of functional work needed for the CTA!

Up next:
- address some largely visual QA in https://github.com/artsy/force/issues/688
- add analytics/AB test with @anipetrov 

This PR:
- bumps artsy-passport for a fix that didn't let you skip onboarding after social auth
- maintains the current filter state (like `?medium=works-on-paper`) through the /payoff and /follow screens

### CODE REVIEW

**Comment by xtina-starr** [comment]
@mzikherman update the specs?

**Comment by mzikherman** [comment]
Just did!

File: apps/artist/routes.coffee [hunk 1]
```diff
@@ -69,7 +70,7 @@ sd = require('sharify').data
   req.user.followArtist req.params.id,
     error: res.backboneError
     success: ->
-      res.redirect "/artist/#{req.params.id}"
+      res.redirect "/artist/#{req.params.id}?#{qs.stringify req.query}"
```
Comment (mzikherman) [inline apps/artist/routes.coffee]: There are where we just pass along any current query params, from /payoff and /follow

File: components/artist_page_cta/templates/register.jade [hunk 2]
```diff
@@ -27,7 +27,7 @@ div.artist-page-cta-overlay__register
   h2
     span Or
 
-  a.auth-signup-facebook.avant-garde-button-black(href="/users/auth/facebook?redirect-to=#{artist.get('href')}/payoff")
+  a.auth-signup-facebook.avant-garde-button-black(href="/users/auth/facebook?skip-onboarding=true&redirect-to=#{afterAuthPath}")
```
Comment (mzikherman) [inline components/artist_page_cta/templates/register.jade]: skip onboarding for social sign up

File: components/artist_page_cta/view.coffee [hunk 3]
```diff
@@ -1,4 +1,5 @@
 Backbone = require 'backbone'
```
Comment (mzikherman) [inline components/artist_page_cta/view.coffee]: The changes here are just about propagating the current query params along to the different links/views.

**Comment by xtina-starr** [comment]
That last failure looks unrelated. I see you kicked it again, fingers crossed. Everything looks good to me so feel free to merge when green.

**Comment by mzikherman** [comment]
🍏 mergey

**Comment by xtina-starr** [comment]
👍

**Pull request merged**